stock_available_unreserved/tests/test_stock_available_unreserved.py

```diff
@@ -214,6 +214,7 @@ def test_stock_search(self):
         no_template = self.env['product.template']
         # Start: one template with three variants.
         # All variants have zero unreserved stock
+
```

Comment:
Why adding an extra line here? PEP8 disregards empty lines in the middle of a method.

Reply:
That was the cherry-pick from jordi. I have removed it.